kDrive/UI/Controller/Files/File List/MultipleSelectionFileListViewModel.swift

```diff
@@ -23,35 +23,48 @@ import kDriveCore
 import kDriveResources
 
 struct MultipleSelectionAction: Equatable {
-    let id: Int
+    private let id: MultipleSelectionActionId
     let name: String
     let icon: KDriveResourcesImages
     var enabled = true
 
+    private enum MultipleSelectionActionId: Equatable {
+        case move
+        case delete
+        case more
+        case deletePermanently
+        case download
+    }
+
     static func == (lhs: MultipleSelectionAction, rhs: MultipleSelectionAction) -> Bool {
         return lhs.id == rhs.id
     }
 
     static let move = MultipleSelectionAction(
-        id: 0,
+        id: MultipleSelectionActionId.move,
```

Comment:
Same for the other ones
```suggestion
        id: .move,
```